packages/frontend/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import styles from './Navigation.module.scss'
+import SakuraParticles from '../SakuraParticles/SakuraParticles'
+import Link from 'next/link'
+
+interface NavProps {
+  title: string
+  page: Page
+}
+
+export enum Page {
+  HOME = 'home',
+  MESSAGES = 'messages',
+  FANART = 'fanart',
+  CLIPS = 'clips',
+  SOUNDBOARD = 'soundboard',
+  CREDITS = 'credits',
+}
+
+interface NavItem {
```

Comment:
Not sure to put this in a separate issue or not, but can't this, together with the formatting, be somehow made into a component? This would make the mapping around line 60 a bit less nested imo.

Reply:
Ah do you mean every nav item?

Reply:
I've decided against this for now since it seems like its more boiler plate than the trouble is worth. I've instead improved some code readability things there and extracted the array to before the component.